src/rules-generator.ts

```diff
@@ -227,7 +225,7 @@ util.methods(Generator, {
   // Intermediate nodes can be "prop" or "$prop" values.
   createValidator: function(schemaName) {
     var schema = this.symbols.schema[schemaName];
-    var validator = {};
+    var validator: {$other?: any} = {}; // todo proper typing
```

Comment:
I don't think that's the correct type...

On Fri, Sep 18, 2015, 12:25 AM Matjaž Lipuš <notifications@github.com>
wrote:

> In src/rules-generator.ts
> <https://github.com/firebase/bolt/pull/54#discussion_r39830037>:
>
> > @@ -227,7 +225,7 @@ util.methods(Generator, {
> >    // Intermediate nodes can be "prop" or "$prop" values.
> >    createValidator: function(schemaName) {
> >      var schema = this.symbols.schema[schemaName];
> > -    var validator = {};
> > +    var validator: {$other?: any} = {}; // todo proper typing
>
> validator is an object with optional $other property of type any. this is
> called type literal in TS.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/firebase/bolt/pull/54/files#r39830037>.
>